sdk/samples/iot/templates/iothub/test-resources-post.ps1

```diff
@@ -65,12 +65,30 @@ Add-AzIotHubDevice `
 -PrimaryThumbprint $fingerprint `
 -SecondaryThumbprint $fingerprint
 
+if ($LASTEXITCODE -ne 0)
+{
+  Write-Host "Adding cert device failed: LAST_ERROR_CODE=${LAST_ERROR_CODE}"
```

Comment:
Would it be helpful to also log some other things like the `$deviceID` or are those considered private?

Reply:
it wouldn't be that useful. All we really need to know is that it failed.